components/builder-depot/src/main.rs

```diff
@@ -65,16 +65,6 @@ fn app<'a, 'b>() -> clap::App<'a, 'b> {
         (@subcommand repair =>
             (about: "Verify and repair data integrity of the package Depot")
         )
-        (@subcommand channel =>
```

Comment:
Good call on removing this stuff, it's almost all dead code since we don't build a depot binary anymore